setup.py

```diff
@@ -43,9 +43,9 @@ def readme():
     package_data={'impala.thrift': ['*.thrift']},
     install_requires=['six', 'bitarray', 'thrift>=0.9.3'],
     extras_require={
-        ":python_version>='3.0'": ["thriftpy2>=0.4.0,<0.5.0",
+        ":python_version>='3.3'": ["thriftpy2>=0.4.0,<0.5.0",
```

Comment:
we could make this 3.4 to match below list of major Python 3 versions, or
add 3.3 below - please let me know what do you think.

thank you.

Reply:
I'm going to leave it as >= 3.0 just in case there are people using older releases of Python (sometimes we see weird things with extended support versions of RHEL, that kinda thing).